pkgs/tools/networking/miniupnpc/default.nix

```diff
@@ -15,8 +15,6 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
-  patches = lib.optional stdenv.isFreeBSD ./freebsd.patch;
```

Comment:
shouldn't we also remove the file from the tree?

Reply:
Oh yes.  Done. :)